### PR TITLE
WIP: Show quotes around testimonials

### PR DESCRIPTION
**Pull request by denisw**
I've added the locale-specific quotes to our translation files.

The reason I marked this as **WIP** is that this probably requires a review by Bea & Masifunde before it can get merged.

### CODE REVIEW

**Comment by sarunast** [comment]
Deploy preview for *masifunde* ready!

Built with commit f943a7f72f9ab9c7f3bb12c1c158e980f7daec69

https://deploy-preview-195--masifunde.netlify.com

File: components/TestimonialList/index.js [hunk 1]
```diff
@@ -22,6 +23,22 @@ const StyledSupporter = Supporter.extend`
   }
 `
 
+const TestimonialText = styled(Markdown)`
+  p:first-child::before {
+    display: inline;
+    /* Use arrow function to ensure translation happens dynamically
+     * (at not just once at component defintion time) */
```
Comment (erikjalevik) [inline components/TestimonialList/index.js]: -> "and not..."
-> "definition time"

Comment (denisw) [reply components/TestimonialList/index.js]: Fixed!

Comment (erikjalevik) [reply components/TestimonialList/index.js]: Forget to push?

Comment (denisw) [reply components/TestimonialList/index.js]: Forgot to stage even 😅 Fixed for good now.

**Pull request merged**